### DISCUSSION TITLE
automatic generation of custom training data as a service

### DISCUSSION THREAD

**Discussion by caco3** [jomjol/AI-on-the-edge-device#1091] [Ideas]
@jomjol @haverland I don't yet understand the process to train new data.

Would it be possible to sinplify the steps so we kind of could provide it as a service using ithub actions.

one way: people could clone a repo, add their images as instructed, then simply wait for the action to complete.
maybe we even could make the commiting easier.

What do you think?

**Comment by ghost** [comment]
I relabel every upload, because it is not easy to label if you not familiar with it. The high precision of the models comes from very accurate labeling. So I'm not a friend of it, to automate it.

**Comment by jomjol** [comment]
I also think, that the training is very specific:
1) Usually labeling is not very accurate and without relabeling we would decrease the quality of the neural networks
2) If we let the user do it completely on their own, they would make dedicated neural networks (so not disturbing the build in ones), but the quality might be bad with bad experience for the users.

**Comment by caco3** [comment]
I was only thinking for the users to do it for themself, not for the general training data.
So the users could play around and it would take workload from @jomjol. So thats more or less  point 2 of @jomjol's thought.

And of course they have to do the labling by themselves with the great tool @haverland wrote.
And they would only do it for themself, so they will retry and do better if it is not good enough :smiling_imp: 

It would need then to read the instructions and be familiar with github, but i expect for many that would be possible if the instructions are good enough.
